src/ducks/pin/PinTimeout.debug.jsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react'
 
+const redAffixStyle = {
+  color: 'white',
+  background: 'crimson',
```

Comment:
Is this a color from cozy-ui? Shouldn't we use a `var(--crimson)`?

Reply:
No, this is a CSS color. I don't think it's necessary to use a variable from cozy-ui here as it is only for debug.